smtlib-backends.cabal

```diff
@@ -1,18 +1,22 @@
-name:          smtlib-backends
-version:       0.1
+name:               smtlib-backends
+version:            0.2
 synopsis:
   Low-level functions for SMT-LIB-based interaction with SMT solvers. 
 
 description:
-  This library provides an extensible interface for interacting with SMT solvers using SMT-LIB. It has built-in support for running solvers as external processes, and the smtlib-backends-z3 package provides a backend that uses inlined calls to Z3's C API.
+  This library provides an extensible interface for interacting with SMT solvers
+  using SMT-LIB. The smtlib-backends-process provides a backend that runs solvers
+  as external processes, and the smtlib-backends-z3 package provides a backend that
+  uses inlined calls to Z3's C API.
 
-license:       MIT
-license-file:  LICENSE
-author:        Quentin Aristote
-maintainer:    quentin.aristote@tweag.io
-build-type:    Simple
-category:      SMT
-cabal-version: >=1.10
+license:            MIT
+license-file:       LICENSE
+author:             Quentin Aristote
+maintainer:         quentin.aristote@tweag.io
+build-type:         Simple
```

Comment:
I didn't really think about this `build-type` field as I mainly reused the `.cabal` file from `simple-smt`. Do we need it here ? Should we change it to something else ?

Reply:
I think some build tools might need the Setup.hs file (e.g. bazel/rules_haskell), so it would be good idea to add it. As for the build type, I'd let cabal checks to say.